packages/pwa/README.md

```diff
@@ -49,76 +59,90 @@ You can choose how the current locale appears (or doesn’t appear) in the URL b
-                    }
-                ]
-            }
-        },
-        {
-            id: 'site-2',
-            l10n: {
-                defaultLocale: 'en-US',
-                supportedLocales: [
-                    {
-                        id: 'en-US',
-                        preferredCurrency: 'USD'
-                    },
-                    {
-                        id: 'en-CA',
-                        preferredCurrency: 'USD'
-                    }
-                ]
-            }
+    {
+        id: 'RefArchGlobal',
+        l10n: {
+            supportedCurrencies: ['GBP', 'EUR', 'JPY'],
+            defaultCurrency: 'GBP',
+            supportedLocales: [
+                {
+                    id: 'de-DE',
+                    alias: 'de',
+                    preferredCurrency: 'EUR'
+                },
+                {
+                    id: 'en-GB',
+                    preferredCurrency: 'GBP'
+                },
+                {
+                    id: 'ja-JP',
+                    preferredCurrency: 'JPY'
+                }
+            ],
+            defaultLocale: 'en-GB'
         }
-    ]
-}
+    }
+]
+```
+
+Optionally, You can then map the site IDs with aliases in `/config/default.js` file and also set default site. If no alias is defined for the site, then IDs are used in URLs. 
+
+```
+\\ /config/default.js
+   
+   defaultSite: 'RefArchGlobal',
+   siteAliases: {
+       RefArch: 'us',
+       RefArchGlobal: 'global',
+       NTOManaged: 'nto'
+   },
 ```
 
+If you set `url.showDefault` to `true` in `/config/default.js` file, then default locale and site are set in the URL of your Retail app. In addition to site alias, you can also configure aliases for your locale in `sites.js` file. If you configured alias, URL will reflect that instead of your locale ID. 
+
+> *Note*: URLs constructed using canonical site and locale ids are still valid URLs even when aliases are used.   
+
+### Manage Multiple B2C Site with Different Domains
+
+It is possible to manage mutliple B2C sites defined in Business Manager and deploy them over different domains using multiple Managed Runtime environments. This is done using a dedicated config file for each environment. For example, your site can be `customer.ca` and `customer.uk`, or `customer-a.com` and `customer-b.ca`. You can also customize the URL patterns further. t is possible to have `customer-1.com/us` and `customer-2.com/?locale=en_US`.
+
+You can deploy bundles to work with multiple domains using different Managed Runtime envioronment, each configured for a separate domain. The configuration mapping is done by means of unique configuration file for each Managed Runtime environment. 
+
+For example, to deploy `customer-1.com` is deployed to envioronment `env-customer-1`, and `customer-2.com` to environment `env-customer-2`, you need to create files `config/env-customer-1.js` and `config/env-customer-2.js` respectively, in place of `config/default.js`. 
+
```

Comment:
I'm wondering if the readme is the right place for this type of documentation. Here we are talking a lot about solutions to problems. But not about the feature itself. I think there is a place for these types of doc's, but probably not in the readme. 

I do however like your verbage "dedicated config files", maybe we can work on that to start documenting the feature. This is rough, but maybe something like:


```
### Dedicated Configuration Files

At project generation a single `default` configuration is created. For most cases a single configuration file will get the job done. But there are times where you want to have a different configuration for a different environment. These environments for example could be, a developers local machine, a remote `production` environment, or a remote `staging` environment. In all these cases you want to use a different configuration for each. 

To achieve this the way in which we resolve what configuration file to use is done in the following way. First, we check to see if there is a config file named after your remote Managed Runtime environment, if none it found we'll then look for a `local` configuration (only if we are running on a developers machine), finally if we still don't find a configuration, we'll load the `default` config file `config/default.js`.

This feature will allow you do these but not limited to things:

- Each developer can have their own configuration, connection to their own Commerce API sandboxes.
- Deploy a single codebase application to multiple environments with their own specific configurations (Multiple B2C Site with Different Domains).

Furthermore, your configurations can be written in any of the following languages, `.js`, `.yml`, `.yaml`, or `.json`. And will be marched based on that priority.  
```

Reply:
I agree with Ben's comment above. The feature alone by itself is not aware of MRT environment or domains until it gets pushed to the platform.  We are replacing a section that documented how the feature works with how they should be used in a platform, which is not relatively true in development per se. A section explaining how the config files could be used sounds right to me.